src/torchcodec/_core/BetaCudaDeviceInterface.h

```diff
@@ -37,7 +37,9 @@ class BetaCudaDeviceInterface : public DeviceInterface {
   explicit BetaCudaDeviceInterface(const torch::Device& device);
   virtual ~BetaCudaDeviceInterface();
 
-  void initializeInterface(AVStream* stream) override;
+  void initializeInterface(
+      const AVStream* stream,
+      const UniqueDecodingAVFormatContext& avFormatCtx) override;
```

Comment:
I just merged #902, so there's going to be merge conflicts. It should be easy to solve: we just rename `initializeInterface()` to `initialize()`, and keep the new param.